docs/quick_start/add_plugins.md

```diff
@@ -0,0 +1,60 @@
+# Add Plugins
+
+Any API Definition can make use of plugins. A plugin is a way to add functionalities to your definition, like rate limit, CORS, authentication, etc...
+
+In this tutorial we will add a [rate limit](/docs/plugins/rate_limit.md) plugin to our definition.
+
+## 1. Add the plugin
+
+Let's create a file called `rate_limit.json` with our new additions to the API definition. We will create a rate limit rula that we can only send `5 req/m`
+ just for us to play around with it.
+
+```json
+{
+  "plugins": [
+    {
+      "name": "rate_limit",
+      "enabled": true,
+      "config": {
+        "limit": "5-M",
+        "policy": "local"
+      }
+    }
+  ]
+}
+```
+
+Now lets update our API definition:
+
+```sh
+http -v PUT localhost:8081/apis/my-endpoint "Authorization:Bearer yourToken" "Content-Type: application/json" < rate_limit.json
```

Comment:
We need to expose `PATCH` method for changing proxies and use `PUT` for update the whole thing